examples/collaborative-editors/README.md

```diff
@@ -0,0 +1,34 @@
+## Purpose
+
+This example can help you get familiar with the API usage of `@otjs/firebase-ace` and `@otjs/firebase-monaco`.
+
+## Prerequisites
+
+If you followed the [installation](https://github.com/Progyan1997/Operational-Transformation#installation) steps successfully, you would have `lerna` and `yarn` installed globally.
+
+Before running this example, it's necessary to build all the packages using `yarn build` in the root (Operational-Transformation) directory.
+
+Also, the **_firebase.json_** in this directory has `database` entry where `rules` point to `".firebase/database.rules.json"` file which is not included with this repository. The user can add their own file with custom rules.
+
+For development purpose, the following can be the content of this file :-
+
+```
+{
+    "rules":{
```

Comment:
Whitespace after `:`

Reply:
Nothing in particular, we are going to squash before merging. So following commit messages yield no value whatsoever, consider using `--fixup` maybe.